src/goto-cc/ld_mode.h

```diff
@@ -40,7 +40,7 @@ class ld_modet : public goto_cc_modet
   /// \brief call ld with original command line
   int run_ld();
 
-  int ld_hybrid_binary(bool);
+  int ld_hybrid_binary(bool, const std::list<std::string> &);
 };
```

Comment:
This is a case where I would include the parameter declarators, as a means of documenting what the arguments would do.

Reply:
Done and documentation for the method added.